linkerd/app/Cargo.toml

```diff
@@ -34,3 +34,4 @@ tokio-stream = { version = "0.1", features = ["time", "sync"] }
 tonic = { version = "0.10", default-features = false, features = ["prost"] }
 tower = "0.4"
 tracing = "0.1"
+pkix = "0.2.3"
```

Comment:
We need to be conservative about pulling in random dependencies of various maintenance statuses... *especially* for our TLS related dependencies.

Can we use [rustls-pemfile](https://docs.rs/rustls-pemfile/latest/rustls_pemfile/), since we already depend on rustls and have a relationship with the maintiners?